ChangeLog

```diff
@@ -74,6 +74,7 @@ Bugfix
      irwir.
    * Enable Suite B with subset of ECP curves. Make sure the code compiles even
      if some curves are not defined. Fixes #1591 reported by dbedev.
+   * Fix partial zeroing in x509_get_other_name. Fixes #2716.
```

Comment:
@ekse If you are OK with it, I'd suggest we add a crediting line like "Found and fixed by YOUR PREFERRED NAME."

Reply:
I added credit to the entry.